src/test/cpp/CMakeLists.txt

```diff
@@ -2,7 +2,28 @@ cmake_minimum_required(VERSION 3.9)
 
 add_executable(cpp_test)
 target_include_directories(cpp_test BEFORE PUBLIC ${CMAKE_CURRENT_SOURCE_DIR} ${CMAKE_CURRENT_SOURCE_DIR}/implementation "${CMAKE_CURRENT_SOURCE_DIR}/generated")
-target_sources(cpp_test PUBLIC pch.cpp activation.cpp test/Async.cpp test/Collections.cpp test/Edge.cpp test/Fast.cpp test/Enum.cpp test/Structs.cpp "${CMAKE_CURRENT_SOURCE_DIR}/generated/module.g.cpp")
+target_sources(cpp_test PUBLIC 
+    "${CMAKE_CURRENT_SOURCE_DIR}/generated/module.g.cpp"
+    pch.cpp
+    activation.cpp
+    test/Async.cpp
+    test/Collections.cpp
+    test/Edge.cpp
+    test/Fast.cpp
+    test/Enum.cpp
```

Comment:
We should alphabetize lists like this so we can more easily pick out changes in the future (though module.g.cpp and pch.cpp should probably be at the top, regardless)